### PR TITLE
C105

### PR DESCRIPTION
**Pull request by vglalitha**
Unit_Testcases

### CODE REVIEW

File: dropwizard-demo/test/com/test/ControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        Assert.assertEquals(expectedTweet, actualTweet);
+    }
+
+    @Test
+    public void testcase_sendTweet_success() throws TwitterException {
+        when(brsConfiguration.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        Twitter twitter = TwitterFactory.getSingleton();
+        String message = "Testing Twitter4J, posting to Twitter programatically hi .";
+        Status status = twitter.updateStatus(message);
+        System.out.println("Successfully updated the status to [" + status.getText() + "].");
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/test/com/test/ControllerTest.java:77]: Assert missing

File: dropwizard-demo/test/com/test/ControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    @Test
+    public void testSearch_finder() throws TwitterException {
+        when(brsConfiguration.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        Twitter twitter = TwitterFactory.getSingleton();
+        Query query = new Query("source:twitter4j yusukey");
+        QueryResult result = twitter.search(query);
+        for (Status status : result.getTweets()) {
+          System.out.println("@" + status.getUser().getScreenName() + ":" + status.getText());
+        }
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/test/com/test/ControllerTest.java:88]: Assert missing, what are u testing here, if there is no assert
what is the use of test case if it never checks/asserts anything?

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]
Address review comments and also u have merge conflicts with master branch
Solve it
then merge it

File: dropwizard-demo/test/com/test/ControllerTest.java (hunk 2 above)
Comment (vglalitha) [reply dropwizard-demo/test/com/test/ControllerTest.java:115]: I am still working on it.

**Review by raj8789 — COMMENTED** [review]
Make changes as @pooja reviewed

**Review by SupriyaPamulapati — APPROVED** [review]

**Pull request merged**